data-model/SDP-data-model-templates/codicological-unit_sdp-template.json

```diff
@@ -219,11 +356,13 @@
       ]
     }
   ],
+  "other_versions": [
```

Comment:
I realized we were missing this field. While it would primarily be used in manuscript objects, I wonder if we would want to provide any URI/identifier that is assigned for the codicological unit by the linked mss aggregator?